java/runtime/src/main/java/org/ray/runtime/AbstractRayRuntime.java

```diff
@@ -143,15 +146,15 @@ public RayObject callPy(String moduleName, String functionName, Object[] args,
     checkPyArguments(args);
     PyFunctionDescriptor functionDescriptor = new PyFunctionDescriptor(moduleName, "",
         functionName);
-    return callNormalFunction(functionDescriptor, args, options);
+    return callNormalFunction(functionDescriptor, args, 1, options);
```

Comment:
maybe add a comment somewhere that void return is not supported for cross-lang calls.

Reply:
added a comment above this line.